packages/block-library/src/post-comments-form/edit.js

```diff
@@ -66,6 +69,16 @@ export default function PostCommentsFormEdit( {
 				),
 				postType
 			);
+			const enableComments = () => setCommentStatus( 'open' );
```

Comment:
I guess this could now be inlined, but it's a matter of style. :)

Reply:
Good point 😄 b95cc0461be3c22a7fa8ab18a3ed30b533fdc3b1